dataans/Cargo.toml

```diff
@@ -45,13 +45,14 @@ web-sys = { version = "0.3", features = [
     "HtmlSelectElement",
 ] }
 futures = { version = "0.3", features = ["std", "alloc"] }
+gloo-timers = "0.3.0"
 
 [workspace]
 resolver = "2"
 members = ["common", "src-tauri"]
 
 [workspace.dependencies]
 time = { version = "0.3" , default-features = false }
-uuid = { version = "1.11", default-features = false }
+uuid = { version = "1.16", default-features = false, features = ["js"]}
```

Comment:
it's better to enable this feature in the crate dependencies (line 11) instead of workspace dependencies. we don't need the `js` feature for the app backend crate